README.md

```diff
@@ -200,9 +211,77 @@ Security Team will need to update the project release node on backdropcms.org to
 inidciate that a release is a security release.
 
 
+The Backdrop Bug Squad
+----------------------
+
+The `Backdrop Bug Squad` consistes of trusted members of the backdrop-contrib
+group who will help other contributors stay on top of minor bug fixes and User
+Interface improvements.
+
+Responsibilities include:
+- Reviewing Pull Requests for eligibility.
+- Merging eligible Pull Requests.
+- Communicating with existing maintainers.
+
+Fixes eligible inlude:
+- Minor Bugs (Fatal errors, PHP notices, validation errors, etc).
+- Minor User Interface changes (typos, spelling, grammar, terminology, etc).
+- Any packaging issue preventing the project from being downloaded and installed
+  via the Project Browser.
+- Any change that could be accepted after a feature freeze.
```

Comment:
Should we add `cross-ports from the original Drupal project` here?

Reply:
sure.

Reply:
added.